js/renovation/ui/toolbar/toolbar.tsx

```diff
@@ -26,4 +24,27 @@ export const viewFunction = ({
   view: viewFunction,
 })
 
-export class Toolbar extends JSXComponent<ToolbarProps>() {}
+export class Toolbar extends JSXComponent<ToolbarProps>() {
+  @Consumer(ConfigContext)
+  config?: ConfigContextValue;
+
+  get fixedProps(): ToolbarProps {
+    const { items } = this.props;
+    const fixedItems = items?.map((item) => {
```

Comment:
Another unclear 'fixed'.
I think it it will be much easier to understand this code with the 'items1' name: I will not guess what 'fixed' means and will immediately investigate the code.

Reply:
Yep, renamed it to toolbarItems